### PR TITLE
Get declaration stats

### PR DESCRIPTION
**Pull request by lera**
returns amount to pay and transactions for requested declaration period, grouped by elster category

### CODE REVIEW

File: lib/graphql/declaration.ts [hunk 1]
```diff
@@ -50,6 +52,51 @@ const SUBMIT_DECLARATION = `
...
+          year: $year
+        ) {
+          amount
+          uncategorized {
+            id
+            amount
+            name
+            purpose
+            valutaDate
+            userSelectedBookingDate
```
Comment (temhota) [inline lib/graphql/declaration.ts:70]: Seems like this should be `selectedBookingDate` and  not `userSelectedBookingDate`
FE is complaining about it

Comment (lera) [reply lib/graphql/declaration.ts:70]: but it was already changed to selectedBookingDate🤔

Comment (lera) [reply lib/graphql/declaration.ts:70]: here https://github.com/kontist/js-sdk/pull/220/files

Comment (lera) [reply lib/graphql/declaration.ts:70]: ah! but not in the method itself 🤦‍♀️

**Pull request merged**